diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index 8c7cc2ec3d9c56a08fca28a28a35ecd47abbc1cb..0000000000000000000000000000000000000000
diff --git a/src/lib/iterable.ts b/src/lib/iterable.ts
index 367003b4a7af75cba1e7ac1b68d5f89d2c402643..4dec97a535b0bb05748523d87ba848ad60cc4154 100644
--- a/src/lib/iterable.ts
+++ b/src/lib/iterable.ts
@@ -17,5 +17,5 @@
  * @public
  */
 export function getFirstFromIterable<T = unknown>(set: Set<T> | Map<any, T>): T {
-	return set.values().next().value
+	return set.values().next().value as T
 }
diff --git a/src/lib/png.ts b/src/lib/png.ts
index b1bbd5703212ddefff61874b4851cd12dde4a049..71a1c16583238272c559730fb6c70162de7d8d7b 100644
--- a/src/lib/png.ts
+++ b/src/lib/png.ts
@@ -164,8 +164,8 @@ export class PngHelpers {
 		const crcBit = new Uint8Array(pHYsData.slice(4, 17))
 		pHYsDataView.setInt32(17, crc(crcBit))
 
-		const startBuf = view.buffer.slice(0, offset)
-		const endBuf = view.buffer.slice(offset + size)
+		const startBuf = view.buffer.slice(0, offset) as ArrayBuffer
+		const endBuf = view.buffer.slice(offset + size) as ArrayBuffer
 
 		return new Blob([startBuf, pHYsData, endBuf], options)
 	}
